Makefile

```diff
@@ -19,7 +19,7 @@ OPEN_PR="true"
 # Force the installation of a Concrete Python version, which is very useful with nightly versions
 # /!\ WARNING /!\: This version should NEVER be a wildcard as it might create some
 # issues when trying to run it in the future.
-CONCRETE_PYTHON_VERSION="concrete-python==2.5.0"
+CONCRETE_PYTHON_VERSION="concrete-python==2.5.1"
```

Comment:
You did this change in another PR: https://github.com/zama-ai/concrete-ml/pull/493/files

Reply:
Yes the other PR targets the release branch and this one main, otherwise the PRs are the same